packages/js/product-editor/src/products-app/product-edit/index.tsx

```diff
@@ -62,7 +59,8 @@ export default function ProductEdit( {
 			return {
 				initialEdits:
 					ids.length === 1
-						? select( 'wc/admin/products' ).getProduct( ids[ 0 ] )
+						? // @ts-expect-error Todo: awaiting more global fix, demo: https://github.com/woocommerce/woocommerce/pull/54146
```

Comment:
Quick question: from this thread (p1735902356485879-slack-C02FL3X7KR6) I understand that the plan is not to disable the errors but to simply leave the TODO comment. This way, if the other PR doesn't fix the errors, we will still be able to catch them in the future.

I guess an alternative approach would to search all inline comments mentioning https://github.com/woocommerce/woocommerce/pull/54146 after it has been merged and remove `@ts-expect-error` to ensure the errors are gone.

I don't have a strong preference between the two approaches, but I think we should be consistent. For example, in https://github.com/woocommerce/woocommerce/pull/54129 we aren't disabling the errors.

Reply:
Sounds good to me. :+1: